content/altinn-studio/app-creation/ui-editor/layout-style/_index.md

```diff
@@ -0,0 +1,132 @@
+        ... 
+        "grid": {
+          "xs": 6,
+        }
+      },
+      {
+        "id": "input-felt-2",
+        "type": "Input",
+        ...
+        "grid": {
+          "xs": 6
+        }
+      }
+}
+
+```
+
+I komponentene over er størrelsen satt til 6 (altså halv bredde) for størrelse `xs`. Siden `xs` er den minste størrelsen tilgjengelig vil dette gjelde fra størrelse `xs` og opp, altså fra `xs` til `xl`.
+Oppsettet over vil gi følgende resultat:
+
+![Halv bredde på skjema](halv-bredde.PNG "Halv bredde på skjema")
+
+Siden man har 12 inndelinger tilgjengelig i sidebredden er det opp til apputvikler å tildele hvor stor andel komponenten skal ha. Om man ønsker at en komponent skal ta 2/12, en annen 6/12 og siste 4/12 så er det mulig.
+Det er viktig å tenke på brukeropplevelsen når man gjør slike endringer, og sidestilling av komponenter bør gjøres med omhu. Komponenter med lengre tekster og beskrivelser blir også fort små og uoversiktelige om disse sidestilles. Skal det benyttes lengre tekster og beskrivelser anbefales det at disse benytter full bredde av skjema.
+
+Om man i eksemepelet over hadde ønsket at komponentene skulle ta hele bredden frem til skjermen ble større enn 960px så kunne man satt opp følgende layout:
+
+```json
+ {
+     {
+        "id": "input-felt-1",
+        "type": "Input",
+        ... 
+        "grid": {
+          "xs": 12,
+          "sm": 12,
+          "md": 6
+        }
+      },
+      {
+        "id": "input-felt-2",
+        "type": "Input",
+        ...
+        "grid": {
+          "xs": 12,
+          "sm": 12,
+          "md": 6
+        }
+      }
+}
+
+```
+
+Da ville komponentene først legge seg sidestilt i det sluttbruker faktisk sitter på en skjerm som oppfyller kravet stilt.
+
+### InnerGrid
+
+Utover det å sette bredde på `grid` i komponenten har vi også lagt til mulighet til å styre `innerGrid`. Å overstyre denne bredden vil typisk være for caser hvor man ønsker at teksten til en komponent skal oppta en større bredde enn input-feltet.
+Du kan tenke på komponenten på denne måten:
+```html
+<Grid id="grid"> 
+    <div>
+        Komponentens tekster
+    </div>
+    <Grid id="innerGrid"> 
+        Komponentens input
+    </Grid>
+</Grid>
+```
+
+Her vil det da være mulig å styre bredden til både ytterste grid og den innerste griden. Eksempel:
+
+```json
+{
+     {
+        "id": "input-felt-1",
+        "type": "Input",
+        ... 
+        "grid": {
+          "xs": 12,
+          "innerGrid": {
+              "xs": 6
+          }
+        }
+      }
+}
+
+```
+
+Her har man satt at komponent griden skal ta hele bredden, mens man begrenser komponentens input til å kun ta halve bredden.
+InnerGrid vil kunne styres for komponentene:
+- Kort svar (Input)
+- Langt svar (TextArea)
+- Filopplaster (FileUpload)
+- Nedtrekksliste (Dropdown)
+- Datovelger (Datepicker)
+
+
+Eksempelet over vil gi følgende output:
+
+![InnerGrid eksempel output](inner-grid.PNG "InnerGrid eksempel output")
```

Comment:
It's a mystery to me why extensions for image-files surprisingly often ends up in terrible upper case. 😃 What kind of tool did you use for images? All extensions should be made lowecase.

Reply:
Snipping tool 😠 fixed.

Reply:
Ahhhh, thanks, nice to know! :)

Reply:
Interesting. git (or possibly vs-code) does not recognize changing file extensions from lowercase to uppercase as an actual change to the file (guessing this is related to windows being case insensitive for file paths).

Looking into how I can fix this 🤓

Reply:
`git config core.ignorecase false` did the trick. Fixed.